app/src/main/java/com/example/xpvehicles/activities/AddVehicleActivity.java

```diff
@@ -45,10 +42,13 @@
 
 import okhttp3.Headers;
 
-public class AddVehicleActivity extends AppCompatActivity {
+public class AddVehicleActivity extends AppCompatActivity implements IndicatorDots {
 
     private static final String TAG = "AddVehicleActivity";
     private static final int CAPTURE_IMAGE_ACTIVITY_REQUEST_CODE = 10;
+    private final String GOOGLE_GEOCODING_API_BASE_URL = "https://maps.googleapis.com/maps/api/geocode/json?";
+    private final String GOOGLE_GEOCODING_API_PARAMETER_KEY = "key";
+    private final String GOOGLE_GEOCODING_API_PARAMETER_ADDRESS = "address";
```

Comment:
constants usually have static associated with them b/c we don't want every instance variable of this class to create these variables.